auth/secp256k1.go

```diff
@@ -21,15 +21,15 @@ const (
 
 // Secp256k1Signer implements the Signer interface using secp256k1 keys
 type Secp256k1Signer struct {
-	privateKey *ecdsa.PrivateKey
+	PrivateKey *ecdsa.PrivateKey
```

Comment:
Why did this change?

Reply:
hmm. Previously, we don't have separate signer for Binance so the signer used this. Since we already have a separated one, this change is not needed. I'll leave it in tact.

Reply:
And why is the private key exposed in the Binance signer?

Reply:
Actually we don't need to exposed tho. Binance Signer aleady have `Sign()` to do the job.